google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineExecutor.java

```diff
@@ -45,8 +45,15 @@ public void onStart(Process process) {
   @Override
   public void cancel() {
     if (process != null) {
-      process.destroy();
-      task.onCancel();
+      // Only destroy the process and signal cancellation if the process hasn't exited
```

Comment:
Check out the info on the issue. We can go over it in more detail in person
too if that's not enough. I'll be in soon.

On Wed, Sep 14, 2016 at 10:17 AM Patrick Flynn <notifications@github.com>
wrote:

> In
> google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineExecutor.java
> <https://github.com/GoogleCloudPlatform/google-cloud-intellij/pull/888#discussion_r78757180>
> :
>
> > @@ -45,8 +45,15 @@ public void onStart(Process process) {
> >    @Override
> >    public void cancel() {
> >      if (process != null) {
> > -      process.destroy();
> > -      task.onCancel();
> > +      // Only destroy the process and signal cancellation if the process hasn't exited
>
> this method cancel() is called by the UI I guess? Why would it be invoked
> multiple times?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/google-cloud-intellij/pull/888/files/0e2d190430bc2d033e13d48f0f480cf211485eb6#r78757180>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABp8QNQU7j6_PYofvcgQ2VDGa1mSSlosks5qqAH7gaJpZM4J7_jY>
> .
>